crates/interpreter/tests/spec.rs

```diff
@@ -24,7 +24,16 @@ use wast::token::Id;
 use wast::{parser, QuoteWat, Wast, WastArg, WastDirective, WastExecute, WastInvoke, WastRet, Wat};
 
 fn test(name: &str) {
-    let path = format!("../../third_party/WebAssembly/spec/test/core/{}.wast", name);
+    test_custom_path(name, "");
+}
+
+fn test_custom_path(name: &str, path: &str) {
```

Comment:
I would instead add parameters to `test()` and update the `test!()` macro. I'll push something on main and then merge it into this branch.